app/delivery/send_to_providers.py

```diff
@@ -82,7 +82,12 @@ def send_sms_to_provider(notification):
             except Exception as e:
                 notification.billable_units = template.fragment_count
                 dao_update_notification(notification)
-                dao_reduce_sms_provider_priority(provider.get_name(), time_threshold=timedelta(minutes=1))
+
+                provider_name = provider.get_name()
+                current_app.logger.info(
+                    "Reducing priority for {provider_name} due to exception raised when calling provider API"
```

Comment:
needs an `f` format string

Reply:
Not essential, but it would be nice if this log line and the log line for slow providers (https://github.com/alphagov/notifications-api/blob/master/app/celery/scheduled_tasks.py#L117) looked similar so we could easily search for both of them to understand the full picture. Or following ben's lead, and make them a metric we can surface in grafana rather than searching in logs 🤔